app.css

```diff
@@ -0,0 +1,77 @@
+﻿body{
+	overflow: hidden;
+	font-family: Verdana, Geneva, sans-serif;
+	text-align: center;
+	margin: 4px;
+}
+table.gridtable{
+	width: 100%;
+	font-size:12px;
+	color:#333333;
+	border-color: #D7D7D7;
+	border-collapse: collapse;
+}
+table.gridtable th {
+	border-width: 0.5px;
+	height: 32px;
+	border-style: solid;
+	border-color: #D7D7D7;
+	background-color: #D3EFFF;
+}
+table.gridtable td {
+	border-width: 0.5px;
+	height: 32px;
+	border-style: solid;
+	border-color: #D7D7D7;
+	background-color: #ffffff;
+}
+table.gridtable td:hover{
+	background-color: #f1f1f1;
+}
+.footer{
+	height: 48px;
+	left: 50%;
+	transform: translateX(-50%);
+	position: absolute;
+	bottom: 4px;
+	user-select: none;
+}
+a{
+	color: black;
+	padding: 8px;
+	text-decoration: none;
+	transition: background-color .3s;
+}
+/* Add a grey background color on mouse-over */
+a:hover:not(.active) {background-color: #D8E9F2;}
+/* search field */
+#search{
+	display: inline-block;
+	width: 84px;
+	font-size: 12px;
+	padding: 8px;
+	border: 1px solid #ddd;
+}
+.loader{
+	position: absolute;
+	left: 46%;
+	top: 40%;
+	border: 16px solid #f3f3f3;
+	border-radius: 50%;
+	border-top: 16px solid #3498db;
+	width: 120px;
+	height: 120px; 
+	-webkit-animation: spin 2s linear infinite; /* Safari */
```

Comment:
Did you test this on Safari?

Reply:
No I didn't - will remove it for now since it's not necessary to include.